modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -2761,6 +2762,24 @@ public void testUpdateObjectAction() throws Exception {
 				"url", "https://onafterdelete.com"
 			).build());
 
+		long objectActionId = objectAction.getObjectActionId();
+
+		AssertUtils.assertFailure(
+			ObjectActionDescriptionException.MustBe75Characters.class,
+			"Description must be less than 76 characters",
+			() -> _objectActionLocalService.updateObjectAction(
+				externalReferenceCode1, objectActionId, false,
+				"equals(firstName, \"João\")", RandomTestUtil.randomString(76),
+				LocalizedMapUtil.getLocalizedMap("Error"),
+				LocalizedMapUtil.getLocalizedMap("Label"), "Name",
+				ObjectActionExecutorConstants.KEY_WEBHOOK,
+				ObjectActionTriggerConstants.KEY_ON_AFTER_ADD,
+				UnicodePropertiesBuilder.put(
+					"secret", "123456"
+				).put(
+					"url", "https://test.com"
```

Comment:
@Aquiiiles please move your test [here](https://github.com/liferay-bpm/liferay-portal/blob/5038eb5a14e472e60408274497ae738621730483/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java#L390) as this is where we are testing exceptions.